integrationtest/docker/jdk8/README.md

```diff
@@ -0,0 +1,8 @@
+## gearpump-docker-for-integration-test
+
+Need Linux Kernel 3.10 (or higher) and Docker 1.7 (or higher). 
+
+1. Install docker ([see also guide](https://github.com/gearpump/gearpump-docker))
+2. Mount Gearpump distribution `sudo mount --bind /path/to/gearpump/dist gearpump`
```

Comment:
@manuzhang I have updated the PR and document, so there is no need to mount Gearpump. The test will mount Gearpump distribution automatically.